Cannot run tests with pytest anymore

Hi everyone,

I was going to work on CircleCI (and this is a confirmation that I probably should have done it sooner) integration, and before doing that I wanted to make sure that `pytest` was still running correctly, but despite having installed all the required libraries in my virtual environment, I'm getting this error:

```
(covid-api) ➜  covid-api git:(master) ✗ pytest
============================================================================================================================= test session starts ==============================================================================================================================
platform darwin -- Python 3.8.1, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: /Users/andrea/Projects/covid-api
collected 0 items / 1 error

==================================================================================================================================== ERRORS ====================================================================================================================================
____________________________________________________________________________________________________________________ ERROR collecting tests/test_health.py _____________________________________________________________________________________________________________________
ImportError while importing test module '/Users/andrea/Projects/covid-api/tests/test_health.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_health.py:1: in <module>
    from fastapi.testclient import TestClient
E   ModuleNotFoundError: No module named 'fastapi.testclient'
```
I've also tried to open a Python shell and import it manually, but I'm getting the same error:
```
(covid-api) ➜  covid-api git:(master) ✗ python
Python 3.8.1 (default, Feb  6 2020, 15:57:25)
[Clang 10.0.1 (clang-1001.0.46.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from fastapi.testclient import TestClient
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'fastapi.testclient'
```
Does anyone know how/when this was broken? Am I doing something wrong?

cc @MatMoore 

Cheers

--- a/covidapi/services/session.py
+++ b/covidapi/services/session.py
@@ -1,6 +1,7 @@
 from sqlalchemy.orm import Session
 from ..db.database import SessionLocal
 
+
 def get_db() -> Session:
     try:
         db = SessionLocal()

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
-fastapi==0.50.0
+fastapi==0.53.0
 pydantic==1.4
 SQLAlchemy==1.3.13
 uvicorn==0.11.3

